server/api/event/register/index.post.ts

```diff
@@ -71,6 +71,28 @@ export default defineEventHandler(async (event) => {
       })
     })
 
+  // check if user meets registration requirements
+  const meetsProgrammeRequirement =
+    data[eventUID]?.registration?.requirements?.programmes?.includes(
+      user?.studyProgram
+    ) ?? true
+
+  const meetsYearRequirement =
+    data[eventUID]?.registration?.requirements?.years?.includes(
+      user?.currentYear
+    ) ?? true
+
+  console.log(meetsProgrammeRequirement, meetsYearRequirement)
```

Comment:
I assume this is for testing. I don't think this should be included in production, even if it is on the server side

Reply:
Good catch. I'll remove it.